hacking/README.md

```diff
@@ -17,7 +17,7 @@ and do not wish to install them from your operating system package manager, you
 can install them from pip
 
     $ easy_install pip               # if pip is not already available
```

Comment:
I think we should do this one separately.  There's questions of  availability that we'd need to figure out. (I checked whether I had get_pip.py on a centos6 box and it wasn't there... then I checked fedora25 and it wasn't installed either).